.vscode/settings.default.jsonc

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
So, I will make this point here since originally, I was going to do a second PR just to discuss it. 

We probably should not have this file at all. By copying over the `settings.default.jsonc` on the devcontainer setup, its essential "locks" the users' settings in place for when their devcontainer was made. The settings hierarchy is Default -> User -> Devcontainer -> Workspace -> Folder. So, by copying over a Folder level settings.json, we are essentially blocking _any_ of the above settings from being used. That means if updates are made to the `devcontainer.json`, the user will _never_ get them because they will not automatically get the new `settings.default.jsonc` (only copies if it does not exist). 

A better practice would not copy over the file or only have the file with examples of how to customize the devcontainer to the user's liking (example would be [the pytest customization](https://github.com/home-assistant/core/pull/73940#discussion_r906004637)]